grunt/tasks/jest.js

```diff
@@ -6,25 +6,87 @@
 
 'use strict';
 
+var assign = require('object-assign');
+var async = require('async');
+var fs = require('fs');
+var glob = require('glob');
 var grunt = require('grunt');
 var path = require('path');
 
+var rootPath = path.resolve('.');
+var buildPath = path.join(rootPath, 'build');
+var tempConfigPath = path.join(buildPath, 'jest-config.json');
+
+var config = require(path.join(rootPath, 'package.json')).jest;
+
+function getCollectCoverageOnlyFrom(callback) {
+  var coverageFrom = config.collectCoverageOnlyFrom;
+  var patterns = Object.keys((config.collectCoverage && coverageFrom) || {});
+  var result = {};
+
+  async.each(patterns, function(pattern) {
+    var options = assign({ nodir: true }, coverageFrom[pattern]);
+    glob(pattern, options, function(err, files) {
+      (files || []).reduce(function(object, key) {
+        object[key] = true;
+        return object;
+      }, result);
+
+      callback(err);
+    });
+  }, function(err) {
+    callback(err, result);
+  });
+}
+
+function getJestConfig(callback) {
+  var rootDir = path.resolve(buildPath, path.resolve(config.rootDir));
+  getCollectCoverageOnlyFrom(function(err, data) {
+    callback(err, assign({}, config, {
+      rootDir: rootDir,
+      collectCoverageOnlyFrom: data,
```

Comment:
The linter wanted a trailing comma on `collectCoverageOnlyFrom: data`. So I added it.